### DISCUSSION TITLE
Organelle Genomes From Long-Read Sequencing

### DISCUSSION THREAD

**Discussion by sibeloncell** [Kinggerm/GetOrganelle#227] [Ideas]
Hello! Would it be possible to adapt GetOrganelle for long-read genome sequences as well? Is that something that is in progress? Thank you.

**Comment by Kinggerm** [comment]
No, and yes.

We have some preliminary tests showing that long-read data alone will not generate organelle genomes as accurately as Illumina at the SNP level, even with HiFi data. So we are working on [Traversome](https://github.com/Kinggerm/Traversome), which can combine short- and long-read data for structure resolution with good SNP-level quality, generating all possible structures if exist. We may incorporate Traversome as a dependency of GetOrganelle in the future.

Also, see #73 , a current solution using long-read data alone was mentioned. Basically, one may use a state-of-the-art long-read assembler to generate an assembly graph and then use `get_organelle_from_assembly.py` to extract the organelle genome. This is what we have found to be not that accurate, especially in the IR region.

**Reply by sibeloncell** [reply]
Thank you so much for your quick reply. I will do so as you recommended.
